Tools/autotest/common.py

```diff
@@ -1301,26 +1302,27 @@ def wait_distance(self, distance, accuracy=5, timeout=30):
                                           % (delta, distance))
         raise WaitDistanceTimeout("Failed to attain distance %u" % distance)
 
-    def wait_servo_channel_value(self, channel, value, timeout=2):
-        """wait for channel to hit value"""
+    def wait_servo_channel_value(self, channel, value, timeout=2, comparator=operator.eq):
+        """wait for channel to exceed value"""
```

Comment:
Comment is incorrect, esp. with the default operator.

Also the case with other comments in here.